python/lib/cloudutils/syscfg.py

```diff
@@ -201,15 +201,15 @@ def check(self):
         if not hostname.isSuccess():
             raise CloudInternalException("Checking hostname ... [Failed]\nPlease edit /etc/hosts, add a Fully Qualified Domain Name as your hostname\n")
         return True
-        
+
 class sysConfigServerRedhat(sysConfigServer):
     def __init__(self, glbEnv):
         super(sysConfigServerRedhat, self).__init__(glbEnv)
         self.svo = serviceOpsRedhat()
-        self.services = [firewallConfigServer(self)]
```

Comment:
@davidjumani I think we should revert the previous behaviour and simply print the statement. Otherwise, env will not deploy. Previously the cloudstack-setup-management would open firewall, now that you've changed it - either (a) document this and fix trillian to open the ports both CentOS/Ubuntu envs, or (b) just print the line for information sake.

Reply:
@rhtyd I was able to deploy a kvm env for basic testing, and everything seems to work. Will look into why it failed this time

Reply:
okay @davidjumani when you confirm we'll be able to merge this

Reply:
It's good to go. Env came up and smoke tests ran successfully!